custom_components/frigate/camera.py

```diff
@@ -69,7 +69,8 @@ def __init__(
         self._cam_name = cam_name
         self._camera_config = camera_config
         self._url = config_entry.data[CONF_URL]
-        self._stream_enabled = self._camera_config["rtmp"]["enabled"]
+        self._stream_enabled = self._camera_config.get("rtmp", {}).get("enabled")
```

Comment:
Might as well use `_attr_is_streaming` for this I think. That way the state will also reflect it.